backend/tests/FuzzTests/ExecutePureFunctions.FuzzTests.fs

```diff
@@ -154,10 +156,20 @@ module Generators =
           return RT.EVariable(gid (), name)
         | RT.TDateTime ->
           let! d = Arb.generate<NodaTime.Instant>
-          return callFn "DateTime" "parse" 0 [ RT.EString(gid (), d.toIsoString ()) ]
+          return
+            callFn
+              "Date"
```

Comment:
Should be DateTime - maybe a merge error.

Reply:
There is a test that is failing after the rebase `"test 1" = $"test {"1"}"`, not sure why looking into it.

Reply:
Fixed